tests/e2e/backup_restore_suite_test.go

```diff
@@ -24,7 +24,7 @@ type VerificationFunction func(client.Client, string) error
 type appVerificationFunction func(bool, bool, BackupRestoreType) VerificationFunction
 
 // TODO duplications with mongoready
-func mongoready(preBackupState bool, twoVol bool, backupRestoreType BackupRestoreType) VerificationFunction {
+func mongoready(preBackupState bool, twoVol bool) VerificationFunction {
 	return VerificationFunction(func(ocClient client.Client, namespace string) error {
 		Eventually(IsDCReady(ocClient, namespace, "todolist"), timeoutMultiplier*time.Minute*10, time.Second*10).Should(BeTrue())
 		exists, err := DoesSCCExist(ocClient, "mongo-persistent-scc")
```

Comment:
yaml files for mongo/mysql where modified (added liveness probe and changed deploy method), but the way we check if apps are ready was not modified. Shouldn't it be also modified?

Reply:
This can be done in separate PR. The way everything is UP should not be changed, just different parts will now exit in appropriate times.

Reply:
not relevant in this PR